### PR TITLE
Help docs

### PR DESCRIPTION
**Pull request by pedromorgan**

### CODE REVIEW

**Comment by pedromorgan** [comment]
OOPS.. wish this could be deleted.. targeted wrong branch..

**Comment by manns** [comment]
@pedromorgan I really appreciate your contributions.
However, such large pull requests are very difficult to handle.

Could you please branch off for each topic (e.g. help or futurization) and send a pull request for each branch?

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-image: debian
+image: arwineap/docker-debian-python3.6
```
Comment (manns) [inline .gitlab-ci.yml]: 1) Pyspread 2 is targeted toward Python3.7

2) I am not using docker. Does this still work?

**Comment by manns** [comment]
@pedromorgan I really appreciate your contributions.
However, such large pull requests are very difficult to handle.

Could you please branch off for each topic (e.g. help or futurization) and send a pull request for each branch?

**Comment by pedromorgan** [comment]
I closed this as I accidentaly targeted master branch instead of python3... in fact I wanted to delete it. forever to avoid this problem.

**Comment by pedromorgan** [comment]
wrong target branch so closing, my mistake,.

**Pull request closed without merging**